packaging/ubuntu-16.04/snapd.postrm

```diff
@@ -107,7 +107,7 @@ if [ "$1" = "purge" ]; then
                 rm -f "$mnt"
             done
         fi
-        find /run/snapd/ns/ \( -name '*.fstab' -o -name '*.user-fstab' \) -delete
+        find /run/snapd/ns/ \( -name '*.fstab' -o -name '*.user-fstab' -o -name '*.info' \) -delete
```

Comment:
I think that `cmd/snap-mgmt/snap-mgmt.sh.in` needs to be updated like this too.

Reply:
Done in cfe86f1ac36f9e469c8f904466ca56bb916fb459